### DISCUSSION TITLE
RTL8710bn - has anyone tried?

### DISCUSSION THREAD

**Discussion by jaymemaurice** [arendst/Tasmota#10961] [Q&A] [answered]
It seems like more and more Tuya devices are coming with rtl8710 chips.
Has anyone tried to pop Tasmota into the Arduino ide and build it for RTL to see how much work would be involved in porting?

**Comment by barbudor** [comment, marked as answer]
1st Tasmota do not build anymore in Arduino IDE. Paltform io is mandatiry
2nd there is a lot of code in Tasmota that is ESP8266 and ESP32 dependent. There is active work on ESP32, none on RTL8710.

A lot of work would be involved and very depending on the quality of the Arduino core for RTL8710.

**Reply by sfromis** [reply]
3rd lots of external libraries are needed to be ported for a Tasmota port having any chance
